app/scripts/metamask-controller.test.js

```diff
@@ -866,6 +866,74 @@ describe('MetaMaskController', () => {
         );
       });
 
+      describe('getHardwareDeviceName', () => {
+        it('should return the correct device name for Ledger', async () => {
+          const deviceName = 'ledger';
+          const hdPath = "m/44'/60'/0'/0/0";
```

Comment:
We use the same `hdPath` in every tests, maybe we could move up one scope above (in the `describe`, WDYT?)

Reply:
yea, i think you give a very good point, let me move the const to describe to reduce the duplilcation.